Client/Frontend/Home/JumpBackIn/Data/JumpBackInDataAdaptor.swift

```diff
@@ -198,8 +155,7 @@ class JumpBackInDataAdaptorImplementation: JumpBackInDataAdaptor, FeatureFlaggab
 
     private func createMostRecentSyncedTab(from clientAndTabs: [ClientAndTabs], completion: @escaping () -> Void) {
         // filter clients for non empty desktop clients
-        let desktopClientAndTabs = clientAndTabs.filter { !$0.tabs.isEmpty &&
-            ClientType.fromFxAType($0.client.type) == .Desktop }
+        let desktopClientAndTabs = clientAndTabs.filter { !$0.tabs.isEmpty }
```

Comment:
Do we want to remove that `== .Desktop` check?

Reply:
Nope, i was testing something and forgot to put it back 🙈